lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -13,12 +13,15 @@ function emailLink({ address, subject = '', body = '' }) {
  */
 export default function ErrorDisplay({ message, error }) {
   let details = '';
-  try {
-    if (error.details) {
+
+  if (typeof error.details === 'object') {
```

Comment:
I think it may only apply to the jinja template (missing scopes) in which case it won't touch this code. However, there was a test that was testing for non-json details so I figured I would add the code to format that test better.